tests.clj

```diff
@@ -82,5 +82,23 @@
       (is (thrown-with-msg? Exception #"nonExistentLicense" 
             (run-license out-file "add" "nonExistentLicense"))))))
 
+(deftest new-scratch-test
+  (let [target-dir (str (fs/temp-dir) "/my-scratch")]
+    (spit (test-file "deps.edn") "{}")
+    (neil (str "new scratch my-scratch :overwrite true :target-dir " target-dir))
+    (is (= (slurp (fs/file "test-resources/new/my-scratch/src/scratch.clj"))
+           (slurp (fs/file (str target-dir "/src/scratch.clj")))))
```

Comment:
Right now I'm using static files in the `test-resources` directory to make the tests work. Unfortunately this will cause the tests to break if the template in `deps-new` is changed.

One way to work around this would be to add a command-line option to swap out the `org.corfield.new/create` function with a mock during testing. That said, I wanted to get some feedback first before adding something like that.

Reply:
Thinking about this some more... maybe a `--dry-run` feature would be better than a mock. Then we can read back the output to check that the `org.corfield.new/create` is being passed the parameters we expect.

Reply:
Added `:dry-run` option.